terraform/modules/provision-aws/eks.tf

```diff
@@ -110,6 +119,22 @@ module "eks" {
         }
       }
 
+      custom_ami = {
+        ami_id = aws_ami.gsa-ise.id
+        enable_bootstrap_user_data = true
+        bootstrap_extra_args = "--container-runtime containerd --kubelet-extra-args '--max-pods=20'"
```

Comment:
I see you planning ahead for the transition to containerd, but given that [it's not yet possible to set the default upstream repo for containerd](https://github.com/GSA/data.gov/issues/3747#issuecomment-1079980390), and it is for dockerd, maybe staying on dockerd is warranted for a while yet.